src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -652,19 +784,32 @@ public void SearchPatternLongPath()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]  // Search pattern with double dots throws ArgumentException
-        public void WindowsSearchPatternWithDoubleDots()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void WindowsSearchPatternWithDoubleDots_Desktop()
```

Comment:
Will we consider porting these changes back to netfx?  What would we do there... quirk it?

Reply:
Yes, we'll quirk. And this will only work in full trust.